server/events.go

```diff
@@ -1643,28 +1701,16 @@ func (s *Server) sendAccConnsUpdate(a *Account, subj ...string) {
 	// Build event with account name and number of local clients and leafnodes.
 	eid := s.nextEventID()
 	a.mu.Lock()
-	localConns := a.numLocalConnections()
-	leafConns := a.numLocalLeafNodes()
-	m := &AccountNumConns{
+	m := AccountNumConns{
 		TypedEvent: TypedEvent{
 			Type: AccountNumConnsMsgType,
 			ID:   eid,
 			Time: time.Now().UTC(),
 		},
-		Account:    a.Name,
-		Conns:      localConns,
-		LeafNodes:  leafConns,
-		TotalConns: localConns + leafConns,
-		Received: DataStats{
-			Msgs:  atomic.LoadInt64(&a.inMsgs),
-			Bytes: atomic.LoadInt64(&a.inBytes)},
-		Sent: DataStats{
-			Msgs:  atomic.LoadInt64(&a.outMsgs),
-			Bytes: atomic.LoadInt64(&a.outBytes)},
-		SlowConsumers: atomic.LoadInt64(&a.slowConsumers),
+		AccountStat: a.accConns(),
 	}
 	// Set timer to fire again unless we are at zero.
-	if localConns == 0 {
+	if m.TotalConns == 0 {
```

Comment:
I am planning on submitting a PR that exclude an account that has JS configured: this is because this event is used to make node "alive" and the absence of those updates cause a server to consider the remote server orphan and switch it to offline. Not sure if that causes a problem with your changes in this PR.

Reply:
I probably should incorporate your change then

Reply:
I can rebase/resolve if yours is committed first. The question I had is: is this PR fundamentally change the fact that the update is sent or not and on existing subject that the server uses to receive conns updates.

Reply:
For me it boils down to if I should even filter accounts when the have no local connections or js.
This is what `IncludeUnused` is about. And I'm wondering if it's needed.
```
// LeafzOptions are options passed to Leafz
type AccountStatzOptions struct {
	Accounts      []string `json:"accounts"`
	IncludeUnused bool     `json:"include_unused"`
}
```